tensorflow/docs_src/install/install_linux.md

```diff
@@ -103,248 +103,203 @@ the specified versions. If upgrading is not possible, then you may still run
+available to everyone on a multi-user system.
 
+Since a system install is not isolated, it could interfere with other
+Python-based installations. But if you understand `pip` and your Python
+environment, a system `pip` install is straightforward.
 
-### Prerequisite: Python and Pip
+See the
+[REQUIRED_PACKAGES section of setup.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/setup.py)
+for a list of TensorFlow packages that `pip` installs or upgrade`.
 
-Python is automatically installed on Ubuntu.  Take a moment to confirm
-(by issuing a `python -V` command) that one of the following Python
-versions is already installed on your system:
 
-  * Python 2.7
-  * Python 3.4+
+#### Install Python and `pip`
 
-The pip or pip3 package manager is *usually* installed on Ubuntu.  Take a
-moment to confirm (by issuing a `pip -V` or `pip3 -V` command)
-that pip or pip3 is installed.  We strongly recommend version 8.1 or higher
-of pip or pip3.  If Version 8.1 or later is not installed, issue the
-following command, which will either install or upgrade to the latest
-pip version:
+On Ubuntu, Python is automatically installed and `pip` is *usually* installed.
+Confirm the `python` and `pip` versions:
 
-<pre>$ <b>sudo apt-get install python-pip python-dev</b>   # for Python 2.7
-$ <b>sudo apt-get install python3-pip python3-dev</b> # for Python 3.n
+<pre class="prettyprint lang-bsh">
+  <code class="devsite-terminal">python -V</code>
+  <code class="devsite-terminal">pip -V  # or: pip3 -V</code>
 </pre>
 
+We *strongly recommend* `pip` or `pip3` version 8.1 or higher. If using a release
+before version 8.1,  upgrade `pip`:
 
-### Install TensorFlow
-
-Assuming the prerequisite software is installed on your Linux host,
-take the following steps:
+<pre class="prettyprint lang-bsh">
+  <code class="devsite-terminal">sudo apt-get install python-pip python-dev   # for Python 2.7</code>
+  <code class="devsite-terminal">sudo apt-get install python3-pip python3-dev # for Python 3.n</code>
+</pre>
 
-  1. Install TensorFlow by invoking **one** of the following commands:
 
-     <pre>$ <b>pip install tensorflow</b>      # Python 2.7; CPU support (no GPU support)
-    $ <b>pip3 install tensorflow</b>     # Python 3.n; CPU support (no GPU support)
-    $ <b>pip install tensorflow-gpu</b>  # Python 2.7;  GPU support
-    $ <b>pip3 install tensorflow-gpu</b> # Python 3.n; GPU support </pre>
+#### Install TensorFlow
 
-     If the preceding command runs to completion, you should now
-     [validate your installation](#ValidateYourInstallation).
+Install one of the available TensorFlow packages:
 
-  2. (Optional.) If Step 1 failed, install the latest version of TensorFlow
-     by issuing a command of the following format:
+<pre class="prettyprint lang-bsh">
+  <code># Select one:</code>
+  <code class="devsite-terminal">sudo pip install tensorflow      # Python 2.7 CPU (no GPU support)</code>
```

Comment:
Should we mention "tf-nightly" here (or earlier)?

(should we in general across all the install pages?)

Reply:
Added here and above (reworded this section a bit)